src/global/mixins/mixins_spacing.scss

```
@@ -1,23 +1,18 @@
 @mixin page-spacing() {
-  padding-inline-start: pxToRem(var(--page-spacing-responsive-left));
-  padding-inline-end: pxToRem(var(--page-spacing-responsive-right));
+  padding-inline-start: pxToRem(var(--page-base-offset-responsive));
+  padding-inline-end: pxToRem(var(--page-base-offset-responsive));
 }
 
 @mixin page-spacing__inner() {
-  @include mq($from: wide--max-content) {
-    width: pxToRem($breakpoint-wide-min);
+  @include mq($from: wide) {
+    width: calc(pxToRem($breakpoint-wide-min) - (2 * pxToRem(var(--page-base-offset-responsive))));
```

would it be possible to wrap pxToRem around the whole calculation like?
```suggestion
    width: pxToRem($breakpoint-wide-min - 2 * var(--page-base-offset-responsive));
```